--- opensmalltalk-vm-sun-v5.0.10/platforms/unix/plugins/SqueakSSL/openssl_overlay.h	2020-07-21 08:25:34.000000000 +0200
+++ p0/opensmalltalk-vm-sun-v5.0.10/platforms/unix/plugins/SqueakSSL/openssl_overlay.h	2020-07-09 11:36:15.459272925 +0200
@@ -164,7 +164,7 @@
 #define sqo_SSL_library_init SSL_library_init
 #define sqo_SSL_load_error_strings SSL_load_error_strings
 
-#define sk_GENERAL_NAME_freefunc void(*)(void*)
+#define sk_GENERAL_NAME_freefunc void(*)(GENERAL_NAME*)
 
 #endif
 
