--- opensmalltalk-vm-sun-v5.0.14/platforms/unix/plugins/XDisplayControlPlugin/Makefile.inc	2020-09-12 15:01:25.000000000 +0200
+++ p0/opensmalltalk-vm-sun-v5.0.14/platforms/unix/plugins/XDisplayControlPlugin/Makefile.inc	2020-09-14 13:06:53.760723212 +0200
@@ -1,9 +1,2 @@
 XCPPFLAGS= -I$(topdir)/platforms/Cross/plugins/FilePlugin -I$(topdir)/platforms/Cross/plugins/SocketPlugin
 XINCLUDES= $(X_INCLUDES)
-XLDFLAGS= $(X_LIBS) -L. -lvm-display-X11
-
-$(TARGET):	libvm-display-X11.so
-
-# note the test below is for handling broken libtool that don't generate the .so extension
-libvm-display-X11.so:
-	test -e ../vm-display-X11/.libs/vm-display-X11 && $(LN_S) ../vm-display-X11/.libs/vm-display-X11 libvm-display-X11.so || $(LN_S) ../vm-display-X11/.libs/vm-display-X11.so libvm-display-X11.so
